client/network/src/service.rs

```diff
@@ -733,25 +733,116 @@ pub struct NetworkWorker<B: BlockT + 'static, H: ExHashT> {
 	/// Senders for events that happen on the network.
 	event_streams: Vec<mpsc::UnboundedSender<Event>>,
 	/// Prometheus network metrics.
-	metrics: Option<Metrics>
+	metrics: Option<Metrics>,
 }
 
 struct Metrics {
+	// This list is ordered alphabetically
+	average_download_per_sec: Gauge<U64>,
+	average_upload_per_sec: Gauge<U64>,
+	import_queue_block_submits: Counter<U64>,
+	import_queue_finality_proofs_submits: Counter<U64>,
+	import_queue_justification_submits: Counter<U64>,
 	is_major_syncing: Gauge<U64>,
+	kbuckets_num_nodes: Gauge<U64>,
+	libp2p_connections: Gauge<U64>,
+	notifications_in: CounterVec<U64>,
+	notifications_out: CounterVec<U64>,
+	num_event_stream_channels: Gauge<U64>,
+	opened_notification_streams: GaugeVec<U64>,
 	peers_count: Gauge<U64>,
+	peerset_num_discovered: Gauge<U64>,
+	peerset_num_requested: Gauge<U64>,
+	random_kademalia_queries: Counter<U64>,
 }
 
 impl Metrics {
 	fn register(registry: &Registry) -> Result<Self, PrometheusError> {
 		Ok(Self {
+			// This list is ordered alphabetically
+			average_download_per_sec: register(Gauge::new(
+				"sub_libp2p_average_download_per_sec", "Average number of bytes downloaded per second",
+			)?, registry)?,
+			average_upload_per_sec: register(Gauge::new(
+				"sub_libp2p_average_upload_per_sec", "Average number of bytes uploaded per second",
+			)?, registry)?,
+			import_queue_block_submits: register(Counter::new(
+				"import_queue_block_submits",
```

Comment:
```suggestion
				"import_queue_blocks_submitted",
```
Sounds better to me. What do you think?